pkg/conduit/pipelines_parser_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conduit
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/conduitio/conduit/pkg/foundation/assert"
+)
+
+func Test_Parser(t *testing.T) {
+	filename, err := filepath.Abs("./test/pipelines1.yml")
```

Comment:
Nitpick: it might be good to have the test yml files named after the use case (e.g. test/pipelines_duplicated.yml)